ci/browser-tests/cypress/integration/activity_spec.js

```diff
@@ -0,0 +1,17 @@
+describe("Activity view", function() {
+    before(() => {
+        cy.dummyLogin("benjamin")
+        cy.selectLanguage("#EN")
+        cy.projectByName("integration test project")
+        cy.get("li a").contains("Detailed design").click()
+    })
+
+    context("Edit Activity", function() {
+        it("can be edited", function() {
+            cy.get("button[class=\"MuiButtonBase-root MuiButton-root " +
```

Comment:
This "calss" approach doesn't seem to be better than going with The "Edit" text?

Reply:
Should insert an id or a data attribute specifically for this case.